README.md

```diff
@@ -3,17 +3,42 @@
 _This package is considered experimental. It is under active/early development,
 and the API is subject to change._
 
-## Developer notes
+## Build Instructions
 
-Run the following command to initialize all of the necessary git submodules.
+1. Clone the repository
+```
+git clone https://github.com/amzn/ion-cli.git
```

Comment:
if you do `git clone --recursive ...` you can skip the step on line 20

Reply:
My problem is that I always clone first and ask questions later. 😛 I usually have the code checked out before I read the instructions, so I figured I'd go with the two-step approach.

Reply:
I know :) it makes the list of instruction smaller.